app/services/idv/usps_mail.rb

```diff
@@ -16,6 +17,12 @@ def any_mail_sent?
       user_mail_events.any?
     end
 
+    def most_recent_otp_expired?
+      return false unless any_mail_sent?
+
+      user_mail_events.first.updated_at < USPS_CONFIRMATION_WINDOW_DAYS.days.ago
```

Comment:
Hmm, that seems like it could get a little bit sticky since the `Event` model is a generic for a number of different user events. So, `sent_at` or `queued_at` may make sense for usps mail events, but maybe not for things like account created events or phone confirmed events.

With that in mind, what is the best way for us to arrange events based on when the events occurred? I would think the `event.created_at` makes sense for that, but went with `updated_at` because that is what I saw in the code for `mail_spammed?`.